ansible/group_vars/all

```diff
@@ -134,24 +134,8 @@ kafka:
   ras:
     port: 8093
   heap: "{{ kafka_heap | default('1g') }}"
-  replicationFactor: "{{ kafka_replicationFactor | default((groups['kafkas']|length)|int) }}"
-  topics:
-    completed:
-      segmentBytes: 536870912
-      retentionBytes: "{{ kafka_topics_completed_retentionBytes | default(1073741824) }}"
-      retentionMS: "{{ kafka_topics_completed_retentionMS | default(3600000) }}"
-    health:
-      segmentBytes: 536870912
-      retentionBytes: "{{ kafka_topics_health_retentionBytes | default(1073741824) }}"
-      retentionMS: "{{ kafka_topics_health_retentionMS | default(3600000) }}"
-    invoker:
-      segmentBytes: 536870912
-      retentionBytes: "{{ kafka_topics_invoker_retentionBytes | default(1073741824) }}"
-      retentionMS: "{{ kafka_topics_invoker_retentionMS | default(172800000) }}"
-    cacheInvalidation:
-      segmentBytes: 536870912
-      retentionBytes: "{{ kafka_topics_cacheInvalidation_retentionBytes | default(1073741824) }}"
-      retentionMS: "{{ kafka_topics_cacheInvalidation_retentionMS | default(300000) }}"
+
+kafka_replicationFactor: "{{ (groups['kafkas']|length)|int }}"
```

Comment:
Is there a reason to change `kafka.replicationFactor` to `kafka_replicationFactor`?

Reply:
I thought flattening was required if we wanted to use it via pureconfig.  Maybe I am still not understanding something here.